src/main/java/eu/dissco/exportjob/service/AbstractExportJobService.java

```diff
@@ -12,30 +13,37 @@
 import java.io.File;
 import java.io.IOException;
 import java.util.List;
+import java.util.Map;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.env.Environment;
 import org.springframework.stereotype.Service;
 
 @Service
 @Slf4j
 @RequiredArgsConstructor
 public abstract class AbstractExportJobService {
 
+  protected static final String ID_FIELD = "dcterms:identifier";
+  protected static final String PHYSICAL_ID_FIELD = "ods:physicalSpecimenID";
+  private static final Map<String, String> extensionMap = Map.of(
```

Comment:
That's an interesting way to do it. It might be a bit clearer to have this be an application property being set based on the profile instead of using a map? But this works